core/primitives/src/trie_key.rs

```diff
@@ -128,6 +128,20 @@ impl TrieKey {
         }
     }
 
+    pub fn get_account(&self) -> Option<&AccountId> {
```

Comment:
`fn account_id` would be a more idiomatic name: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter